config.toml

```diff
@@ -3,7 +3,6 @@ refresh_rate = "1s"
 
 [server]
 address = "127.0.0.1:8000"
-#url = "https://rustypaste.shuttleapp.rs"
```

Comment:
Second thought, since this config is not documented elsewhere maybe we can leave this in as a comment but change to URL to https://example.com or something like that.

What do you think?

Reply:
yep, this probably makes sense. But I'd use something like:

```
https://paste.example.com
or
https://example.com/paste
```

In config.toml and examples/, right? Please let me know which one of the 2 above is better.

Reply:
yup, sounds good. I think `config.toml` is better.

Reply:
I meant one of the 2 from the URLs. ;-) I think we can add it to config.toml and to the examples/html_form.toml.

Reply:
Ah, let's go with `paste.example.com` with both files then.